components/indexer.py

```diff
@@ -112,15 +117,13 @@ def execute(self) -> None:
             intake_main_motor.set(self.intake_motor_speed)
         else:
             intake_main_motor.set(0)
-        for i, motor in enumerate(
-            self.indexer_motors, len(self.all_motors) - len(self.indexer_motors) - 1
-        ):
+        for i, motor in enumerate(self.indexer_motors, 1):
```

Comment:
This line could probably still use a comment explaining how the start value is to account for the offset between indexer_motors and first ball

Reply:
Done, PTAL.